ogusa/parameters.py

```diff
@@ -484,11 +484,14 @@ def read_tax_func_estimate(self, tax_func_path):
                     flag = 1
             except KeyError:
                 pass
-            if self.tax_func_type != dict_params['tax_func_type']:
-                print('Model tax function type is not ' +
-                      'consistent with tax function parameter ' +
-                      'esimtates')
-                flag = 1
+            try:
+                if self.tax_func_type != dict_params['tax_func_type']:
+                    print('Model tax function type is not ' +
+                          'consistent with tax function parameter ' +
+                          'esimtates')
+                    flag = 1
+            except KeyError:
+                pass
```

Comment:
@jdebacker I just happened to look over this when I saw you were updating `parameters.py` and have two small suggestions:
- typo in estimates
- it might be more straghtforward to do:
    ```
    if self.taxfunc_type != dict_params.get('tax_func_type', None):
    ...
    ```
    to handle the case where 'tax_func_type' isn't in `dict_params`

Reply:
Thanks for the suggestions!  Will do.

Reply:
Actually, @hdoupe while your suggestion avoids the `KeyError` for missing items in the dict, I wanted behavior that was compatible with older version of the tax parameter estimates so that if the key was missing, a flag *is not* raised.  Therefore, I've switched back to the `try: except:`   phrases.

Reply:
Aha! Sorry for leading you astray. Looks good to me.